integration/helpers/create-fixture.tsx

```diff
@@ -166,10 +167,23 @@ export async function createAppFixture(fixture: Fixture) {
        * @param waitForHydration Will wait for the network to be idle, so
        * everything should be loaded and ready to go
        */
-      goto: async (href: string, waitForHydration?: true) => {
-        return page.goto(`${serverUrl}${href}`, {
-          waitUntil: waitForHydration ? "networkidle0" : undefined,
-        });
+      goto: (async (href: string, opts?: any) => {
```

Comment:
I didn't actually change any of this. Did I miss something in a merge somewhere?